cassandra/src/main/dist/config/tables.cql

```diff
@@ -21,6 +21,7 @@ CREATE TABLE freki.id_to_name (
   type varchar,
   creation_time timestamp,
   name varchar,
+  meta varchar,
```

Comment:
This column should be named to reflect what it actually contains (the description).